src/modules/function/declaration.rs

```diff
@@ -271,13 +277,74 @@ impl TranslateModule for FunctionDeclaration {
 impl DocumentationModule for FunctionDeclaration {
     fn document(&self, meta: &ParserMetadata) -> String {
         let mut result = vec![];
-        result.push(format!("## `{}`", self.name));
+        result.push(format!("## `{}`\n", self.name));
+
+        let references = self.create_reference(meta, &mut result);
+
         result.push("```ab".to_string());
         result.push(self.doc_signature.to_owned().unwrap());
         result.push("```\n".to_string());
         if let Some(comment) = &self.comment {
             result.push(comment.document(meta));
         }
+
+        if let Some(references) = references {
+            for reference in references {
+                result.push(reference);
+            }
+            result.push("\n".to_string());
+        }
+
         result.join("\n")
     }
 }
+
+impl FunctionDeclaration {
+    #[cfg(debug_assertions)]
```

Comment:
This annotation ensures the new code is only compiled for debug builds, so we don't need to test the path for `target/debug`.  The complementary `#[cfg(not(debug_assertions))]` annotation below hard codes the return value for release builds.

Reply:
how we can compile a debug build? `cargo build` locally is enough? just to check the CI if needs something to not compile that

Reply:
Use `cargo build` for a debug build, `cargo build --release` for a release build.